coralproject/elkhorn
# [BOCOUP] Coral bocoup

This pr introduces an endpoint and module that gathers aggregation data from askd and publishes it to a series of files. The form and content of these files was originally developed for the Coral - Bocoup election form interactive but are generic enough to be useful in a variety of visualizations and other reports on form submissions.

How to test:

The functionality in this branch is supported by these branches:
https://github.com/coralproject/shelf/tree/coral-bocoup
https://github.com/coralproject/cay/tree/coral-bocoup-groups

With these group pulled
1) Create a form in cay
2) Check Group By on at least one multiple choice question
3) Check Include in Groups for one or more questions
4) Hit elkhorn: http://localhost:4444/publish/aggregations/form/{formId}
5) Check that the digest and group files are written in accordance with the elkhorn config.
